cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Babbage/CommandLineArg/ConfigFile.hs

```diff
@@ -0,0 +1,37 @@
+module Test.Cardano.Db.Mock.Unit.Babbage.CommandLineArg.ConfigFile (
+  checkConfigFileArg,
+)
+where
+
+import Cardano.Mock.ChainSync.Server (IOManager)
+import Data.Text (Text)
+import Test.Cardano.Db.Mock.Config (CommandLineArgs (..), babbageConfigDir, withCustomConfigAndLogs)
+import Test.Tasty.HUnit (Assertion)
+
+mkCommandLineArgs :: Bool -> CommandLineArgs
+mkCommandLineArgs hasConfigFile =
+  CommandLineArgs
```

Comment:
It may be less verbose to have a default `CommandLineArgs` and write here only the values that are different, using a notation of this form
```
defaultCommandLineArgs {claHasConfigFile =  claHasConfigFile, ...} 
```

Reply:
good spot I've actually got one already too `initCommandLineArgs`!!